src/Compilers/Core/Portable/Compilation/DataFlowAnalysis.cs

```diff
@@ -100,6 +100,11 @@ public abstract class DataFlowAnalysis
         /// </summary>
         public abstract ImmutableArray<ISymbol> UnsafeAddressTaken { get; }
 
+        /// <summary>
+        /// The set of local functions that are called.
```

Comment:
The member name is `used` but this comment says `called`. This is an important distinction because local functions can be used but not called. This occurs when doing method group to delegate conversions. 

```cs
void Outer() 
{
  void Inner() { } 
  Action local = Inner; // Used, not called
}
```

Should make this consistent.